lib/convert.js

```diff
@@ -167,7 +169,11 @@
         }
 
         // Write an image of format <format> to STDOUT
-        args.push(format + ":-");
+        if (format === "jpg" && useFlite && settings.useOptimalJPEG) {
```

Comment:
currently Save for web just enabled optimal or not, do you think we would want 

jpg:encode-fast or,
jpg:encode-precomputed

( I assume, perhaps incorrectly that the other option jpg:encode-default is the same as jpeg:- )